apps/src/sites/studio/pages/raceInterstitial.js

```diff
@@ -0,0 +1,56 @@
+import $ from 'jquery';
+import experiments from '@cdo/apps/experiments';
+
+$(document).ready(function () {
```

Comment:
Nit: all these functions could be written using ES6 fat arrows, which we tend to prefer where possible, i.e.
```
$(document).ready(() => {
```